OK-43922 OK-43786 OK-43848, feat: enhance perps trading with cross margin ratio, iOS input improvements and validation fixes

Minnzen opened the pull request.

Minnzen committed:
feat: add cross margin ratio calculation functionality

- Update ClosePositionModal to use correct position size
- Remove close operation column from PerpPositionsList
- Introduce and display cross margin ratio in PerpAccountPanel
- Add useMMR hook to calculate cross margin ratio

Minnzen committed:
feat: enhance trading input components with iOS InputAccessoryView support

- Add InputAccessoryView with Done button for iOS numeric inputs in TpslInput
- Implement smart InputAccessoryView detection in TradingFormInput
- Optimize PriceInput with useMemo for actions and decimal-pad keyboard
- Add keyboardType decimal-pad to SizeInput for better mobile experience
- Improve mobile input accessibility with keyboard dismiss functionality

Minnzen committed:
fix: correct position size calculation and input validation in ClosePositionModal

- Update ClosePositionModal to use correct position size (positionSize instead of position.szi)
- Fix amount calculation logic to ensure proper positionSize usage
- Add leading zero validation in perpsUtils for both size and price inputs
- Improve input validation consistency across trading forms

revan-zhang commented:
### :tada: **Snyk checks have passed. No issues have been found so far.**

:white_check_mark: **security/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/rayston92/pr-checks/9789e179-25a6-4969-8b61-a3730ae97b77?refs=74f5efebb7fae6d89e80b51fe3e7392b5a64cd89&source=prChecksComment)

:white_check_mark: **license/snyk** check is complete. No issues have been found. [(View Details)](https://app.snyk.io/org/rayston92/pr-checks/9789e179-25a6-4969-8b61-a3730ae97b77/license?refs=74f5efebb7fae6d89e80b51fe3e7392b5a64cd89&source=prChecksComment)

originalix commented on packages/kit/src/views/Perp/components/TradingPanel/inputs/TpslInput.tsx:
```diff
@@ -1,23 +1,81 @@
+    <>
+      <Input {...inputProps} inputAccessoryViewID={accessoryViewId} />
+      {platformEnv.isNativeIOS ? (
+        <InputAccessoryView nativeID={accessoryViewId}>
```
DepositModal 也一起处理了吧

Minnzen replied:
对噢  好

Minnzen committed:
Merge branch 'x' into fix/perps-trading

Minnzen committed:
refactor: consolidate InputAccessoryView component usage in DepositWithdrawModal

- Export InputAccessoryDoneButton from TradingFormInput for reuse
- Remove unnecessary keyboard display logic and simplify code structure

Minnzen committed:
Merge branch 'fix/perps-trading' of https://github.com/OneKeyHQ/app-monorepo into fix/perps-trading
